src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,7 +1,35 @@
 package core.basesyntax;
 
+import java.io.File;
+
 public class WorkWithFile {
+    private static final String SUPPLY = "supply";
+    private static final String BUY = "buy";
+    private static final String TOTAL = "result";
+    private static final char CHAR = ',';
+    private static final String SEPARATOR = System.lineSeparator();
+    private static final int WORD_INDEX = 0;
+    private static final int NUMBER_INDEX = 1;
+
     public void getStatistic(String fromFileName, String toFileName) {
+        int supplySum = 0;
```

Comment:
this method is already incorrect - it does too many things

you should have separate method for calculating report